Tasks/DotNetCoreCLIV2/pushcommand.ts

```diff
@@ -12,6 +12,7 @@ import * as pkgLocationUtils from 'packaging-common/locationUtilities';
 import { getProjectAndFeedIdFromInputParam, logError } from 'packaging-common/util';
 
 export async function run(): Promise<void> {
+    tl.loc('DeprecatingDotnet2_2');
```

Comment:
intentional? `tl.loc` just retrieves a message and returns it.

Reply:
i copy/pasta'ed it from the restore task, so I thought it should be there.  I can take it out.